CHANGELOG.MD

```diff
@@ -12,6 +12,9 @@ All notable changes to this project will be documented in this file.
 -->
 ## [Unreleased]
 
+### Fixed
+- apply marker filtering to all polygons inside feature
```

Comment:
Code lijkt ok, maar ik ben niet helemaal mee wat er hier gebeurt.

Reply:
leaflet geeft na click op een feature de hele feature mee, dit feature object bevat een geometry object. Hierin zit een coordinates object wat een array is van:
-2 dimensionale array (x & y coördinaat van alle punten) ==> polygoon
-nested array van polygonen

Eerst werd er enkel maar voor de eerste entry in de coordinates array gekeken of een marker hierin voorkwam. 
Nu wordt er mapping gedaan om alle polygonen uit coordinate object op te halen en wordt er gekeken welke markers hier zich bevinden in een van deze polygonen.